src/css/_keyframes.scss

```diff
@@ -1,8 +1,50 @@
-@keyframes slidein {
-  from {
-    margin-bottom: 100%;
+@-webkit-keyframes tracking-in-expand {
```

Comment:
Keyframes are the COOLEST THING IN THE WORLD!!! I can see myself spending a massive amount of time in this realm.